.github/workflows/BuildAndRelease.yml

```
@@ -7,12 +7,13 @@ on:
   pull_request:
     branches:
       - develop
-    types: [closed]
+    types: [ closed ]
 
   workflow_dispatch:       # 手動実行も可能。
 
 permissions:
   contents: write      # リリース作成とリポジトリ参照に必要な最小権限
+  variables: write     # リポジトリ変数の更新に必要
```

actionlint (1.7.12)

[error] unknown permission scope "variables". all available permission scopes are "actions", "artifact-metadata", "attestations", "checks", "contents", "deployments", "discussions", "id-token", "issues", "models", "packages", "pages", "pull-requests", "repository-projects", "security-events", "statuses"

(permissions)